<launch>
    <node name="pub_kpt_pch_raw" pkg="sitl_dvrk" type="pub_kpt_pch_raw.py" output="screen">
        <param name="calib_dir"           type="str"     value="L2R"/>
        <param name="model_path"          type="str"     value="/home/$(env USER)/dt2_dataset/kpts/pch_results_t5/model_final.pth"/>
        <param name="model_score_thr"     type="double"  value="0.8"/>
        <param name="window_size"         type="int"     value="25"/>
        <param name="mad_thr"             type="double"  value="3"/>
        <param name="kpt_score_thr"       type="double"  value="0.1"/>
    </node>
    <node name="pub_kpt_pch_cp" pkg="sitl_dvrk" type="pub_kpt_pch_cp.py" output="screen">
        <param name="tf_path"             type="str"     value="/home/$(env USER)/aruco_data/base_tfs.yaml"/>
        <param name="frame_id"            type="str"     value="ecm_left"/>
        <param name="tip_child_frame_id"  type="str"     value="dt2_psm1_tip"/>
        <param name="jaw_child_frame_id"  type="str"     value="dt2_psm1_jaw"/>
    </node>
    

    <?ignore
    
    ?>
</launch>